internal/backends/collection.go

```diff
@@ -211,9 +212,13 @@ func (cc *collectionContract) Explain(ctx context.Context, params *ExplainParams
 }
 
 // CollectionStatsParams represents the parameters of Collection.Stats method.
-type CollectionStatsParams struct{}
+type CollectionStatsParams struct {
+	Refresh bool // TODO https://github.com/FerretDB/FerretDB/issues/3483
```

Comment:
Should setting refresh call `full vacuum` or is it related to calling `analyze`?

Which handler would have Refresh = false/true then?

Reply:
This PR is a subset of #3461; we need to merge this one first because a few linked issues are easy for contributors to pick up.

`Stats` with `Refresh: true` should call `ANALYZE`. But that's a different issue, yeah. Maybe that will not be needed.

Reply:
Cool :+1:

So for now, every handler will have `Refresh: true` set since it calls `Analyze`.

Reply:
No, for now, let's change nothing yet.

Reply:
Actually, let me remove it from that PR…